bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolderRenderer.java

```diff
@@ -449,8 +449,8 @@ protected Rectangle computeTrim (int part, int state, int x, int y, int width, i
 					if (!parent.simple && !parent.single && (state & SWT.SELECTED) != 0) {
 						width += curveWidth - curveIndent;
 					}
-					y = y - ITEM_TOP_MARGIN;
-					height = height + ITEM_TOP_MARGIN + ITEM_BOTTOM_MARGIN;
+					y = y - ITEM_TOP_MARGIN - parent.selectionHighlightBarThickness;
+					height = height + ITEM_TOP_MARGIN + ITEM_BOTTOM_MARGIN + parent.selectionHighlightBarThickness;
```

Comment:
Can we draw the bar in the Item Margin instead of increasing the item height? Currently this causes the height of the TabItem header to increase/decrease based on the bar thickness.
AFAICS the current implementation in Eclipse Dark theme also draws in the available space - https://github.com/eclipse-platform/eclipse.platform.ui/blob/master/bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java#L584

Reply:
ok, I'll try that.